### DISCUSSION TITLE
Is there the method to segment cranial nerve and cerebrovascular automatically and visually using tractography technique of 3D-Slicer based on MONAILabel?

### DISCUSSION THREAD

**Discussion by ulphypro** [Project-MONAI/MONAILabel#1410] [Q&A]
Dear all members!

Perheps is there the method to segment cranial nerve and cerebrovascular automatically and visually using tractography technique of 3D-Slicer based on MONAILabel?

**Comment by diazandr3s** [comment]
Hi @ulphypro,

Thanks for posting this question.
Do you mean using diffusion MR images to segment the cranial nerve? Would you please provide more details?

**Reply by ulphypro** [reply]
Dear @diazandr3s 

Thank you for answering my question.


I want to segment whole brain composed as cerebrovascular, cranial nerve or brain tumor as 'Auto Segmentation' using tractography technique in 3D-Slicer. 

Is it possible to segment brain as 'Auto Segmentation' using tractography technique in 3D-Slicer?

Is there the method?

**Reply by diazandr3s** [reply]
Hi @ulphypro,

MONAI Label has the algorithms for you to train on different image modalities and labels. 
Currently, we don't have a model for you to run auto segmentation out of the box on this type of modality. 
Have you considered creating ground truth labels to train an AI model for this?
